quality-assurance/safety-assurance-guidance.md

```diff
@@ -1,44 +1,47 @@
 
-* Error (Data Integrity)
-* Delay
+# Overview of approach
 
-This is centered around asking *What if* questions to determine safety risks and the likelihood and safety impact of those risks occurring.
+Our approach is closely related to our successful Developer Threat Modelling process which is used as part of our security lifecycle.
 
-## Bowtie Method
+The test engineer in each delivery team is responsible for enforcing our safety practices.  They run a lightweight process of tagging work items as safety-related and then creating barriers and mitigations to safety hazards.
 
-> The bowtie method is a risk assessment method that can be used to analyse and communicate risk scenarios. The method takes its name from the shape of the diagram that you create, which looks like a man’s bowtie. A bowtie diagram mainly does two things. First of all, a bowtie gives a visual summary of all plausible incident scenarios that could exist around a certain hazard. Second, the bowtie represents what an organisation does to control those scenarios by identifying safety barriers.
+If teams need more information about the safety context of their work, they will request that it is provided by product owners.  
 
-See [Bowtie in four easy steps](https://www.youtube.com/watch?v=PHbLQWqojC8)
+# Identifying safety-related projects
 
-## How should we use it
+A team’s product owner is accountable for establishing whether their current delivery is safety-related.
 
-### At the start of a project
+If the delivery is safety-related, then the team’s “definition of done” must contain a statement to the effect that safety modelling must have been carried out for every PBI (work item)
 
-* Carry out a bowtie analysis of the project to identify potential areas of safety that need to be considered (there are a number of safety assurance experts who can help in carrying out bowtie analysis).
+# Process – Safety Modelling
 
-### During Sprints
+When developing safety-related software, the following process must be followed.
 
-* Tag relevant PBI's with a `safety` tag along with details of how the safety risk can be mitigated. If it is not clear whether there is a safety impact use a `safety pending` tag until this ambiguity is resolved.
-* Consider testing the risk mitigation. There will be a balance between effort required to test against the likelihood and impact.
-* Let the PO make final decisions on the safety impact/consequence. The PO will engage with other SME's to determine if the safety risk is acceptable. This may  
-  mean that although there is a potential safety risk that risk is deemed manageable or may require remedial action. This detail should be captured in a lightweight 
-  manner within the relevant PBI.
- 
-## General Principles 
+1. If a PBI is considered as potentially relevant to safety, it must be marked in the work tracking system with the standard tag ‘Safety Pending’.
```

Comment:
With markdown you can use "1." for all points and the numbers are auto-created

Reply:
done